src/d3a/models/strategy/settlement/strategy.py

```diff
@@ -200,6 +207,15 @@ def event_offer_traded(self, strategy: BidEnabledStrategy,
             strategy.state.decrement_unsettled_deviation(
                 trade.offer_bid.energy, market.time_slot)
 
+    def get_unsettled_deviation_dict(self, strategy: "BidEnabledStrategy") -> Dict:
+        return {
+            "unsettled_deviation_kWh": {
+                format_datetime(time_slot): strategy.state.get_unsettled_deviation_kWh(
```

Comment:
As this is d3a-api-client user facing, i suggest to use the `DATE_TIME_FORMAT` instead of the UI format. Thanks!

Reply:
will change it thanks!